product_docs/docs/pgd/5/taskbook/administration/rolling_database_upgrades.mdx

```diff
@@ -0,0 +1,342 @@
+Deleting files from new pg_xact                             ok
+Copying old pg_xact to new server                           ok
+Setting oldest XID for new cluster                          ok
+Setting next transaction ID and epoch for new cluster       ok
+Deleting files from new pg_multixact/offsets                ok
+Copying old pg_multixact/offsets to new server              ok
+Deleting files from new pg_multixact/members                ok
+Copying old pg_multixact/members to new server              ok
+Setting next multixact ID and offset for new cluster        ok
+Resetting WAL archives                                      ok
+Setting frozenxid and minmxid counters in new cluster       ok
+Restoring global objects in the new cluster                 ok
+Restoring database schemas in the new cluster               ok
+Copying user relation files                                 ok
+Setting next OID for new cluster                            ok
+Sync data directory to disk                                 ok
+Creating script to delete old cluster                       ok
+Checking for extension updates                              notice
+
+Your installation contains extensions that should be updated
+with the ALTER EXTENSION command.  The file
+    update_extensions.sql
+when executed by psql by the database superuser will update
+these extensions.
+
+
+Upgrade Complete
+----------------
+Optimizer statistics are not transferred by pg_upgrade.
+Once you start the new server, consider running:
+    /usr/pgsql-15/bin/vacuumdb --all --analyze-in-stages
+
+Running this script will delete the old cluster's data files:
+    ./delete_old_cluster.sh
+
+pg_upgrade complete, performing BDR post-upgrade steps
+------------------------------------------------------
+Collecting old cluster control data                         ok
+Collecting new cluster control data                         ok
+Checking LSN of new cluster                                 ok
+Starting new cluster with BDR disabled                      ok
+Connecting to new cluster                                   ok
+Creating replication origin (bdr_bdrdb_rb69_bdr2)           ok
+Advancing replication origin (bdr_bdrdb_rb69_bdr2, 0/1F4... ok
+Creating replication origin (bdr_bdrdb_rb69_bdr1)           ok
+Advancing replication origin (bdr_bdrdb_rb69_bdr1, 0/1E8... ok
+Creating replication slot (bdr_bdrdb_rb69_bdr1)             ok
+Creating replication slot (bdr_bdrdb_rb69)                  ok
+Creating replication slot (bdr_bdrdb_rb69_bdr2)             ok
+Stopping new cluster
+```
+
+### 10 - Modify the Postgres service
+
+The Postgres service on the system is configured to start the old version 
+of Postgres (PG14). We need to modify the `postgres.service` file to start the new 
+version (PG15). This can be done using sed to replace the old version 
+number "14" with "15" throughout the file. Once changed, we can tell the 
+systemd daemon to reload the configuration. On the target node, run:
```

Comment:
What? Shouldn't this be handled properly by the packages? At least on Debian (sorry, my bias), you can run multiple instances and different major versions in parallel. I would strongly recommend against manually fiddling with `.service` files there.